src/converters/lintConfigs/rules/ruleConverters.ts

```diff
@@ -532,11 +565,23 @@ export const ruleConverters = new Map([
     ["rxjs-no-nested-subscribe", convertNoNestedSubscribe],
     ["rxjs-no-redundant-notify", convertNoRedundantNotify],
     ["rxjs-no-sharereplay", convertNoShareReplay],
+    ["rxjs-no-subclass", convertNoSubclass],
     ["rxjs-no-subject-unsubscribe", convertNoSubjectUnubscribe],
     ["rxjs-no-subject-value", convertNoSubjectValue],
+    ["rxjs-no-tap", convertNoTap],
```

Comment:
The most correct way to convert this rule and `no-do` would be to convert it to `ban-operators` (as [**`no-tap`** is deprecated](https://github.com/cartant/eslint-plugin-rxjs/blob/dd5e8cf3ba24685f1cfba97eff5be95171b8b053/source/rules/no-tap.ts#L22) however, the `ban-operators` might already be declared in `tslint.json` and then it would give an error, I believe. Thoughts on this, @JoshuaKGoldberg?